src/main/java/uk/gov/hmcts/darts/event/mapper/EventHandlerMapper.java

```diff
@@ -29,7 +29,7 @@ public EventHandlerEntity mapFromEventMappingAndMakeActive(EventMapping eventMap
     }
 
     private String getHandlerMappingOrNullHandler(String handler) {
-        return handler != null ? handler : DartsEventNullHandler.class.getSimpleName();
+        return handler != null && !handler.equalsIgnoreCase("") ? handler : DartsEventNullHandler.class.getSimpleName();
     }
```

Comment:
how about we simplify using Guava `Strings.isNullOrEmpty()`?

Reply:
I was going to make the same suggestion as @mario-paniccia , but would suggest `StringUtils.isNotBlank()` from `apache.commons` which covers null, empty and blank cases.